mitiq/interface/conversions.py

```diff
@@ -20,6 +20,51 @@ class CircuitConversionError(Exception):
     pass
 
 
+register_to_dict: Dict[Tuple[str, str], Callable[[Any], Circuit]]
```

Comment:
Given we define two dictionaries one for `to` and other for `from`, why do we need to store the direction in the key?